spec/active_record_query_counter_spec.rb

```diff
@@ -184,6 +187,16 @@
         end
       end
     end
+
+    it "keeps a count of rollbacks for each transaction trace" do
```

Comment:
Can you also add to the test on line 134 to add a rolled back transaction to that test case:

```ruby
    it "is not enabled outside of a count_queries block" do
      ActiveRecord::Base.transaction { true }
      ActiveRecord::Base.transaction { raise ActiveRecord::Rollback }
      expect(ActiveRecordQueryCounter.transaction_count).to eq nil
      expect(ActiveRecordQueryCounter.rollback_count).to eq nil
      expect(ActiveRecordQueryCounter.transaction_time).to eq nil
      expect(ActiveRecordQueryCounter.transactions).to eq nil
    end
```